### ISSUE TITLE
does not adjust for printing

### ISSUE THREAD

**Issue by revansx** [enterprisemediawiki/MasonryMainPage#15]
Works great except when a user tries to print. Is there some CSS to add to Mediawiki:Print.css that would help?

**Comment by darenwelsh** [comment]
Do you mean how the blocks get split between page breaks? I think this is a general problem for MediaWiki since we've also had issues with images and tables split across pages. You can try the CSS properties `page-break-after`, `page-break-before`, and `page-break-inside`. Maybe this could be built into the masonry blocks.

**Comment by revansx** [comment]
No. it's the width. When a user on a widescreen monitor goes to print, the print (typically portrait) only shows the left side of the screen. The right side is cropped off in the print preview/print. It can be fixed if the user prints in landscape and plays with the scale, but fundamentally, the blocks (which are intended to "re-arrange themselves" don't re-arrange themselves for the printed page. I was just wondering if there is something that can be done in Mediawiki:Print.css that might trigger the blocks to automatically re-arrange themselves for the printed page like they do for the screen page.

**Comment by darenwelsh** [comment]
Do you have a screen capture you can share? When I preview the print in both portrait and landscape, it does not seem to have width issues.
